src/_gettsim_tests/test_piecewise_polynomial.py

```diff
@@ -0,0 +1,45 @@
+"""
+Tests for `piecewise_polynomial`
+"""
+import numpy as np
+from _gettsim.piecewise_functions import (
+    get_piecewise_parameters,
+)
+
+
+def test_get_piecewise_parameters_all_intercepts_supplied():
+    params = {
+        "type": "piecewise_linear",
+        "progressionsfaktor": True,
+        0: {
+            "lower_threshold": "-inf",
+            "upper_threshold": 2005,
+            "rate_linear": 0,
+            "intercept_at_lower_threshold": 0.27,
+        },
+        1: {
+            "lower_threshold": 2005,
+            "upper_threshold": 2021,
+            "rate_linear": 0.02,
+            "intercept_at_lower_threshold": 0.5,
+        },
+        2: {
+            "lower_threshold": 2021,
+            "upper_threshold": 2041,
+            "rate_linear": 0.01,
+            "intercept_at_lower_threshold": 0.8,
+        },
+        3: {
+            "lower_threshold": 2041,
+            "upper_threshold": "inf",
+            "rate_linear": 0,
+            "intercept_at_lower_threshold": 1,
+        },
+    }
+    expected = np.array([0.27, 0.5, 0.8, 1])
+    actual = get_piecewise_parameters(
```

Comment:
Better call by name, not by value -- I'd have to guess what the second and third arguments do. Also, `eink_st_tarif` seems misleading?